xs2a-adapter-rest-impl/src/main/resources/static/main.js

```diff
@@ -455,4 +436,40 @@ function addTooltips(e) {
         helper.setAttribute("class", "mdl-tooltip mdl-tooltip--top");
     }
 }
+function persist() {
+    fetch("v1/aspsps/persist", {
+        method: "POST"
+    })
+        .then(response => {
+            if (!response.ok) {
+                throw Error(response.statusText);
+            }
+            success();
+        })
+        .catch(error => {
+            console.log(error);
+            operationFailed();
+        })
+}
+
+function paginate(data) {
+    let dataLength = data.length;
+    let step = 10;
+    let current = 0;
+    let button = document.querySelector(".show-more");
+    let total = document.querySelector(".total");
+
+    addPage();
+    total.innerHTML = dataLength;
+
+    function addPage() {
+        for (limit = current + (step > dataLength ? dataLength : step); current < limit; current++) {
```

Comment:
Haven't you tried to use `Math.min()` (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/min) instead of `(step > dataLength ? dataLength : step)`?

Reply:
yep, that seems better looking